nominees/notesmaster-.json

```diff
@@ -0,0 +1,55 @@
+{
+  "name": "NotesMaster ",
+  "aliases": [
+    "Always learning, Together"
+  ],
+  "description": "Working in partnerships to support the sustainable access to digital learning resources.",
+  "website": "https://notesmaster.com/",
+  "license": [
+    {
+      "spdx": "CC-BY-SA-4.0",
+      "licenseURL": "https://notesmaster.com/termsofuse"
```

Comment:
The terms of use linked here do not include any explicit mention to `CC-BY-SA-4.0`, nor does it include any mention that user content is licensed that way. As a matter of fact, it does not explicitly mention anywhere under which terms is user content licensed.

Reply:
@notesmaster kindly include the phrase below on your website as stated in the attached document for us to be able to pass the open licensing indicator:

```
Digital Content Licence. All content created on NotesMaster by its users and administrators is given the following attribution:
Attribution-ShareAlike 4.0 International (CC BY-SA 4.0)

```

Reply:
@nathanbaleeta This will be available on the platform tomorrow. I will notify you when shown.

Reply:
@nathanbaleeta This is now available on the platform. Go to the homepage and scroll down: https://notesmaster.com/